changelog/hasAlwaysInlines.dd

```diff
@@ -0,0 +1,4 @@
+Always inline `pragma(inline, true)` functions regardless of -inline compiler switch.
+
+Marking a function with `pragma(inline, true)` will now cause it to be always inlined
+when called, and a compile error will result if it cannot be inlined.
```

Comment:
I think it should be mentioned that this might not apply to functions with `pragma(inline,true)` specified in the function body. BTW: is this case going to be deprecated?

Reply:
Another notable change is that failing to inline is no longer an error, but only displays a warning now.

Reply:
done

Reply:
done

Reply:
Thanks. It still says an error occurs, though, while actually only informational warnings are shown (as specified below).